### PR TITLE
Retailer case study

### PR DESCRIPTION
**Pull request by dstodolny**
### Motivation

- [Related story](http://...)

TODO: write a motivation.

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by gregdorward** [comment]
- [ ] The headings in red currently use "font-family: Sentinel A,Sentinel B,Georgia,serif", whereas the Zeplin says that these should use "font-family: ProximaNova-Bold"

**Comment by gregdorward** [comment]
In the table at the bottom of the content, the rows need to align, as shown in the design, where each row is 50px in height.

This branch:
screen shot 2017-03-21 at 11 09 33

Design:
screen shot 2017-03-21 at 11 10 14

**Comment by gregdorward** [comment]
Just a couple of minor issues remaining in the checklist above

**Comment by gregdorward** [comment]
One more that I missed...

- [ ] The following h2 element needs to be increased to 40px in size: 
class="style__content__title___1Bd6Z _fonts__fontM2___3sSED _fonts__sansSerif___3iE3M"

With this change, we'll also need to address the padding in the below area to make sure the "Tell us more" button has sufficient padding at the bottom

Also, this one still needs amending:

- [ ] This header needs to be updated to "font-family: ProximaNova-Bold" class="style__listBox__heading___eRROr _fonts__fontS2___3XzgT _fonts__sansSerif___3iE3M"

**Comment by gregdorward** [comment]
To provide extra context, can we change the alt text for the image "class="style__techGraphImage___Pv91c"" to something along the lines of this please?

alt="Image showing the tecnology stack used in this project, including Jenkins, Amazon Web Services, React, Express.js, GraphQL"

File: site/pages/our-work/case-study/retailer/index.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        <ul className={styles.content__list}>
+          <li className={styles.content__list__element}>A shorter feedback loop</li>
+          <li className={styles.content__list__element}>Team-wide responsibility for each decision</li>
+          <li className={styles.content__list__element}>Elimination of waste</li>
+          <li className={styles.content__list__element}>Driven determination and focus in order to deliver value to customers, faster</li>
+          <li className={styles.content__list__element}>A sped-up delivery, over four years early</li>
+        </ul>
+      </div>
+      <div>
+        <img src={techGraphImage} className={styles.techGraphImage} alt="Tech graph" />
```
Comment (gregdorward) [inline site/pages/our-work/case-study/retailer/index.js]: To provide extra context, can we change the alt text for this image to something along the lines of this please?

alt="Image showing the tecnology stack used in this project, including Jenkins, Amazon Web Services, React, Express.js, GraphQL"

Comment (dstodolny) [inline site/pages/our-work/case-study/retailer/index.js]: Our linter throws error with that:
```
error    Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don't need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop 
```

**Comment by gregdorward** [comment]
Aside from my comment above, I'm happy with this one, so it just needs an official review at this stage.

This shouldn't be merged into Staging until the contact us piece is finished though, so we don't have a non-functioning button on the page

File: site/pages/our-work/case-study/retailer/index.js (hunk 1 above)
Comment (gregdorward) [inline site/pages/our-work/case-study/retailer/index.js]: Could we just remove "Image" from the start of the sentence then. The default img tag doesn't really provide a very detailed description as is

**Comment by dstodolny** [comment]
Here it is: https://github.com/redbadger/website-honestly/pull/420 🐎

**Comment by gregdorward** [comment]
@dstodolny I've just noticed with the style__techGraphImage___Pv91c image, we have left side padding of 20 pixels applied, but 0 on the right hand side. Could we apply this padding to both sides please? On a mobile viewport, we don't need any padding on either side, which is what we're already doing in this case

**Comment by gregdorward** [comment]
Just needs a review and can then be merged along with the contact us slice ticket

**Comment by chris-brookes** [comment]
@dstodolny - The main page content (not including the header and footer) should have a max width of 1400px, with the content equally split over 12 columns. See grid in the image attached.

screen shot 2017-03-24 at 11 28 41

**Comment by chris-brookes** [comment]
@dstodolny - The full 12 column design should still be shown at the smallest desktop view (980px width). Come and see me if you need me to explain any further, and I can show you what I mean.

**Pull request merged**